doc/source/whatsnew/v1.1.0.rst

```diff
@@ -585,6 +585,8 @@ I/O
   unsupported HDF file (:issue:`9539`)
 - Bug in :meth:`~DataFrame.to_parquet` was not raising ``PermissionError`` when writing to a private s3 bucket with invalid creds. (:issue:`27679`)
 - Bug in :meth:`~DataFrame.to_csv` was silently failing when writing to an invalid s3 bucket. (:issue:`32486`)
+- :func:`read_parquet` now supports an s3 directory (:issue:`26388`)
```

Comment:
can you review the doc-strings to see if they need updating (e.g. may need a versionadded tag)

Reply:
[Parquet Docs strings](https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.read_parquet.html) indicate we already supported this I think? I updated the whatsnew and added an example in docs strings.